src/opm/input/eclipse/Schedule/Well/WellConnections.cpp

```diff
@@ -217,6 +217,17 @@ namespace Opm {
         }
     }
 
+    bool WellConnections::underWREVInjMultMode() const {
+        // TODO: if the first connection is under WREV, all the connections should be under WREV
+        // how we can guarantee that?
+        // and hesitate to add extra member to this class
```

Comment:
Please clarify what the code currently does. Will the value actually  be the same for all connections?

Reply:
Yes, it should be.

Reply:
At the end, with more investigation, I put the mode to the Well class. So we do not need to check the mode per connection.